packages/shorebird_cli/lib/src/validators/validators.dart

```diff
@@ -30,14 +30,21 @@ extension Display on ValidationIssueSeverity {
 /// A (potential) problem with the current Shorebird installation or project.
 @immutable
 class ValidationIssue {
-  const ValidationIssue({required this.severity, required this.message});
+  const ValidationIssue({
+    required this.severity,
+    required this.message,
+    this.fix,
+  });
 
   /// How important it is to fix this issue.
   final ValidationIssueSeverity severity;
 
   /// A description of the issue.
   final String message;
 
+  /// Fixes this issue.
+  final Future<void> Function()? fix;
```

Comment:
```suggestion
  final FutureOr<void> Function()? fix;
```
minor but couldn't this be `FutureOr` since not all fixes will need to be async?

Reply:
Done